eclipse-theia/theia-blueprint
# update to @theia 1.13.0

#### What it does

Update to latest @theia,  v1.13.0

#### How to test

Build this PR and confirm that it's still possible to successfully package and run packaged version of `Blueprint`. 

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
